tests/jvm/src/test/scala/scala/meta/tests/semanticdb/InteractiveSuite.scala

```diff
@@ -110,7 +110,7 @@ class InteractiveSuite extends FunSuite {
     // typer hijacking is not working as expected with interactive.Global.
     expectedPrevious,
     compat = List(
-      "2.13.4" -> expectedLatest,
+      "2.13.5" -> expectedLatest,
```

Comment:
I would leave `2.13.4` key still here. The test will fail otherwise.

Reply:
We should also add `2.13.5` key, sorry if I wasn't clear about it. It should look like:
```
      "2.13.4" -> expectedLatest,
      "2.13.5" -> expectedLatest,
```

Reply:
I added the small change myself to unblock.